service/rtc/sfu.go

```diff
@@ -444,9 +412,14 @@ func (s *Server) InitSession(cfg SessionConfig, closeCb func() error) error {
 			us.mut.Unlock()
 
 			call.iterSessions(func(ss *session) {
-				if ss.cfg.SessionID == us.cfg.SessionID {
+				// We skip sending the track to the current session (the one sending it) and to any session
+				// that hasn't finished initializing its tracks (they are still connecting).
+				// This is to avoid queuing duplicate tracks. The call of call.mut.Lock() is needed to guarantee
+				// we won't be missing any tracks.
```

Comment:
can you explain: how do we ensure the connecting session gets the track (after it's finished connecting)?

Reply:
It was the second path I was having trouble tracking -- I can't see where we set it and defer...  But I see that we only enter the `handleTracks` after dc is open.